verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -0,0 +1,96 @@
+import MaterialTable from 'material-table';
+import Paper from '@material-ui/core/Paper';
+import React from 'react';
+import axios from 'axios';
+
+interface ListResponse {
+    sources: Source[],
+    nextPage: number,
+    total: number,
+}
+
+interface Origin {
+    url: string;
+    license: string;
+}
+
+interface Field {
+    name: string;
+    regexp: string;
+}
+
+interface Parsing {
+    fields: Array<Field>;
+}
+
+interface Automation {
+    name: string;
+    tag: string;
+    active: boolean;
+    scheduleExpression: string;
+    parsing: Parsing;
+}
+
+interface Source {
+    _id: string;
+    name: string;
+    origin: Origin;
+    format: string;
+    automation: Automation;
+}
+
+interface SourceTableState {
+    tableRef: any,
+    url: string,
+}
+
+export default class SourceTable extends React.Component<{}, SourceTableState> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            tableRef: React.createRef(),
```

Comment:
This (and the other tableRefs) can be removed. I'm removing it from the other table in an in-progress PR because it turned out to be unnecessary.

Reply:
Excellent, thanks.